tensorflow/contrib/cmake/tf_cc_ops.cmake

```diff
@@ -120,3 +120,41 @@ list(REMOVE_ITEM tf_cc_srcs ${tf_cc_test_srcs})
 
 add_library(tf_cc OBJECT ${tf_cc_srcs})
 add_dependencies(tf_cc tf_cc_framework tf_cc_ops)
+
+set (pywrap_tensorflow_lib "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_BUILD_TYPE}/pywrap_tensorflow_internal.lib")
```

Comment:
Checking my understanding: is this the import `.lib` for `pywrap_tensorflow_internal.dll`? If so, do we need to include it in the PIP package so that third parties can write extension modules? (And can you add a comment here to explain this as well?)